client/blocks/stats-navigation/index.js

```diff
@@ -46,7 +39,7 @@ class StatsNavigation extends Component {
 
 	state = {
 		isPageSettingsPopoverVisible: false,
-		modules: {
+		pageModules: {
```

Comment:
The initial state defined here isn't the same as how it's used: https://github.com/Automattic/wp-calypso/pull/77866/files#diff-c48db36d0f71bd2871975d28a9aaeca7d92c3bcfc04e227e1eddee46109dd285R69

Am i missing anything?

Reply:
Fixed in [5fcd450](https://github.com/Automattic/wp-calypso/pull/77866/commits/5fcd450a2d7f64b721f405a355c67fc138c6f9ef).